core/modules/nfv_monitor.h

```diff
@@ -36,9 +37,14 @@ class NFVMonitor final : public Module {
   CommandResponse CommandGetSummary(const bess::pb::EmptyArg &arg);
 
   double GetTailLatency() {
-    sort(per_core_latency_sample_.begin(), per_core_latency_sample_.end());
-    size_t idx = ceil(0.99 * per_core_latency_sample_.size());
-    return per_core_latency_sample_[idx];
+    std::vector<uint64_t> latency_copy_(per_core_latency_sample_.size());
```

Comment:
1. This initialization will create per_core_latency_sample_.size() samples in the vector. Later, you push N more samples to the vector again. This is not what 'copy' means, right?
2. xx_ naming is just for class members. Please use it only for class members.

Reply:
I tested on my server, and found the vector was not empty after the initialization. Can you please check this on your side?